src/component/handlers/drag/DraftEditorDragHandler.js

```diff
@@ -23,6 +23,7 @@ const getUpdatedSelectionState = require('getUpdatedSelectionState');
 const nullthrows = require('nullthrows');
 
 import type SelectionState from 'SelectionState';
+const isEventHandled = require('isEventHandled');
```

Comment:
For required modules, we still stick to `require` internally at FB. We use `import` syntax when importing Flow types only.